go/engine/login_provision.go

```diff
@@ -983,11 +988,9 @@ func (e *loginProvision) chooseGPGKeyAndMethod(m libkb.MetaContext) (*libkb.GpgP
 	// find any local private gpg keys that are in user's key family
 	matches, err := e.matchingGPGKeys(m)
 	if err != nil {
-		if _, ok := err.(libkb.NoSecretKeyError); ok {
```

Comment:
i think we should keep this and figure out why we aren't telling the user to do this...

Reply:
@patrick it's just an empty if block though, keeping it doesn't do anything -- what code do you want me to put inside it?  A log statement?  I'm happy to file a bug for someone else to look at the reasoning behind it, but I'm not going to do any deep debugging as part of this PR.

Reply:
this looks like a "TODO" comment left by someone that never evolved into actual code that would "tell the user they need to get a gpg key" onto this device.